src/features/events/components/groupedEventsCard.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import moment from 'moment'
+import { FlexColumn } from '../../../components/grid'
+import { media } from '../../../theme/media'
+import { dateFormat, itemsToLoad } from '../../../constants'
+import EventListingCard from './eventListingCard'
+
+const DateGroupHeading = styled.h2`
+  margin: 1rem 0;
+
+  ${media.tablet`
+    display: none;
+  `};
+`
+
+const GroupedEventsCards = ({ event, index, events, toLoad }) => {
+  let header
```

Comment:
See comment about `let animate`. A similar approach could be taken here by having a simple `const` ternary.